plugins/onepad_legacy/onepad.cpp

```diff
@@ -223,10 +216,6 @@ EXPORT_C_(void) PADsetLogDir(const char *dir)
 
 EXPORT_C_(void) PADclose()
 {
-    while (!ev_fifo.empty())
```

Comment:
Doesn't the new type queue need to be flushed here too?

Reply:
You know, I was largely copying greg's code from onepad over to onepad legacy, and he didn't flush it there. Still, this is on closing the plugin, and I'm not sure it's absolutely necessary. 

Actually, I'm pretty sure if the plugin was closed, that it'd have to call PADopen if it started again, and that resets the queue, so we're probably fine there.

Reply:
I guess. Do note that plugin may unload after closing it, but then it should just run the destructor and free the queue anyway.